templates/bootstrap.sh

```diff
@@ -463,7 +463,7 @@ else
     fi
 
     # Run the build script command in the service specified in BUILDKITE_DOCKER_COMPOSE_CONTAINER
-    echo "+++ $BUILDKITE_COMMAND_ACTION (in Docker Compose container)"
+    echo "~~~ $BUILDKITE_COMMAND_ACTION (in Docker Compose container)"
```

Comment:
We only want the "Running command" line to expand if no other "---" or "+++" sections are supplied by the build command output, because most of the time it's just the output of the command execution:

```
~~~ Running command...
./script/something
--- Running specs...
...
```

We make sure that "Running command..." or similar is expanded if no other sections are open in the Buildkite frontend, and I've verified that the section is expanded for the all the scenarios I've listed in the original pull request message, including the docker compose ones. :-)